hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -203,10 +203,9 @@ protected void incrementCounter(Statistic objectsRead) {
   public OzoneFSOutputStream createFile(String key, boolean overWrite,
       boolean recursive) throws IOException {
     incrementCounter(Statistic.OBJECTS_CREATED);
-    try {
-      OzoneOutputStream ozoneOutputStream = bucket
-          .createFile(key, 0, replicationType, replicationFactor, overWrite,
-              recursive);
+    try (OzoneOutputStream ozoneOutputStream = bucket
+        .createFile(key, 0, replicationType, replicationFactor, overWrite,
+            recursive)) {
```

Comment:
This stream is returned to the caller for writing.  It cannot be closed in this method.

Reply:
@adoroszlai Thanks for the review. You are right! I fixed this in the latest patch.